src/solidity/utils.ts

```diff
@@ -24,52 +24,65 @@ export function findImports(data: ImportFile): string[] {
   return result;
 }
 
-// when importing files from parent directory solc requests sifferent file name
-// e.g. for "../dir/file" solc requests file named "dir/file"
-// solidifyName returns file name that will  be requested by solc
-export function solidifyName(fileName: string): string {
-  return fileName.split("./").pop()!;
+interface ImportTreeNode extends ImportFile {
+  uri: string;
+  imports: { uri: string; url: string }[];
```

Comment:
What does the 'url' and 'uri' mean in this context? More clear and useful would be `{ absolute: string, relative: string }`. This would also retain all the information about imports.

Reply:
URI and URL in this context mean the same exact thing as in Resolver-Engine in general. uri of the import is the shortcut name (eg: github:cymerrad/file.sol) and url is the resolved one